### DISCUSSION TITLE
A function that increases the value of a specific field by 1 each time a message is sent

### DISCUSSION THREAD

**Discussion by KIYONG-KWON** [dannagle/PacketSender#322] [Q&A]
Is the function installed in the latest version that is currently being distributed? If not, is there a way you can recommend?

**Comment by dannagle** [comment]
This does not exist. There are macros that can manipulate the data, but there isn't a macro that is a simple incrementing counter. 

You can get close. You could use `hh:mm:ss.zzz` as your `{{TIME}}` macro format, and then you would have a continuously incrementing value down to the millisecond in your data field. 

If you want to manipulate the GUI in an automated way, the fields do support ctrl+NUM to easily jump to them.
